roles/rsyslog/templates/input_relp.j2

```diff
@@ -0,0 +1,59 @@
+input(name="{{ item.name }}"
+      type="imrelp"
+      port="{{ item.port | d(20514) | int }}"
+      maxDataSize="{{ item.max_data_size | d(logging_max_message_size) | int }}"
+{% if item.use_cert | default(true) %}
+{%   if item.ca_cert is defined %}
+{%     set __cacert = item.ca_cert %}
+{%   elif item.ca_cert_src is defined %}
+{%     set __cacert = __rsyslog_config_dir + '/' + item.ca_cert_src | basename %}
+{%   else %}
+{%     set __cacert = __rsyslog_config_dir + '/es-ca.crt' %}
```

Comment:
input relp uses `es-ca.crt`?

Reply:
Sorry, copy&paste error... :p (Actually, there's no chance to go there since the logging role had failed prior to this template lookup if both ca_cert and ca_cert_src are not configured.)